### PR TITLE
docs(examples): add Magic Link + JWT example

### PR DESCRIPTION
**Pull request by jstafman**
Closes: #

- [ ] Docs
- [ ] Tests

Re-submission of #2149

### CODE REVIEW

File: examples/magic-link-with-jwt/.gitignore [hunk 1]
```diff
@@ -0,0 +1,307 @@
+node_modules
```
Comment (MichaelDeBoey) [inline examples/magic-link-with-jwt/.gitignore:1]: Please keep this to the bear minimum like we have in the `.gitignore` file in the templates

https://github.com/remix-run/remix/blob/main/templates/remix-ts/.gitignore

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks for working on this!

Unfortunately this doesn't follow [the instructions](https://github.com/remix-run/remix/tree/main/examples#contributing) for contributing examples. Please read those and update this example to be in line with our guidelines.

**Comment by jstafman** [comment]
@kentcdodds - I actually didn't realize that there were separate contribution guidelines for examples. I'll be sure to review. Thanks for pointing that out.

**Comment by machour** [comment]
Closing this PR as we haven't received a response from the original author.

**Pull request closed without merging**